vendor/github.com/grpc-ecosystem/grpc-gateway/runtime/convert.go

```diff
@@ -261,7 +261,7 @@ func EnumSlice(val, sep string, enumValMap map[string]int32) ([]int32, error) {
 }
 
 /*
-	Support fot google.protobuf.wrappers on top of primitive types
```

Comment:
Maybe we should not manually update things from `vendor`. 
You can file a PR directly on `github.com/grpc-ecosystem/grpc-gateway`.

Reply:
ok